bots/images/scripts/lib/atomic.install

```diff
@@ -166,23 +166,38 @@ class AtomicCockpitInstaller:
         """ Install the latest cockpit RPMs in our container"""
         rpm_args = []
         for package in self.rpms:
-            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package:
+            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package or 'libssh' in package:
```

Comment:
This is old code, but let me ask anyway: What does it do?  Why does it prepend "/host" only to some packages? Is there a more general way to decide which packages need to be modified, maybe by checking the filesystem to see which files actually exist?

Reply:
`self.rpms` here contains all the cockpit rpms, and now the rpms supplied by `--extra`. And basically this filters out the ones that get into the `cockpit/ws` container inside of the image.

So the docker cockpit-ws image only has a few packages in it, and basically the update_container only freshens those. It does not contain cockpit-ssh. However in order to install cockpit-ssh (container in cockpit-dashboard on centos-7/rhel-7) libssh is needed, which is not installed. 

However maybe this should be a separate install step, since `update_container` was really only meant to freshen the packages and not install new packages.

Reply:
Oh, I misread the code on first look.  This is actually fltering the packages from self.rpms into rpm_args, not just prepending "/host" to some of them, and not others.  Thanks!